tests/integration/test_package_managers.py

```diff
@@ -160,3 +123,82 @@ def test_packages(
         )
         log.info("Compare checksums of files in source vendor folder")
         assert files_checksums == expected_files_checksums
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        # Test case checks fetching retrodep dependencies, generating environment vars file,
+        # building image with all prepared prerequisites and printing help message for retrodep
+        # app in built image
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/retrodep.git",
+                ref="c3496edd5d45523a1ed300de1575a212b86d00d3",
+                packages=({"path": ".", "type": "gomod"},),
+                check_vendor_checksums=False,
+                install_app="retrodep",
```

Comment:
two nitpicks for `install_app`

1. Should it be a separate pytest param, not part of TestParameters? The command is never relevant for non-e2e tests
2. Could it be something like a `check_cmd: list[str]` instead (and set it to `["retrodep", "--help"]`)? Not all e2e tests will want to call `<some_app> --help`

Reply:
ack great idea